app/assets/v2/js/grants/_new.js

```diff
@@ -165,7 +165,8 @@ Vue.mixin({
         'grant_type': form.grant_type,
         'categories[]': form.grant_categories,
         'network': form.network,
-        'region': form.region
+        'region': form.region,
+        'opt_out_clr': form.opt_out_clr ? '1' : '0'
```

Comment:
could we stick to true / false as opposed to 1/0 ?

Reply:
`true` / ` false` / `null` / `undefined` are sent as string values to backend, so it will always be evaluated as python `True`

that's why i did `bool(int(request.POST.get('opt_out_clr')))` as a way to convert '1' / '0' to python boolean

what do you think?

Reply:
I'd still say stick to true/false! I know we had this discussion a few months back cause I had the same question -> and we agreed to use true / false -> which is I ended up sticking to that 

https://gitcoincore.slack.com/archives/G01LRE6F39A/p1606144083009000

Reply:
done